### ISSUE TITLE
[timestampTrade] Python error when performing sync task

### ISSUE THREAD

**Issue by slabking** [stashapp/CommunityScripts#170]
Hello! I'm encountering a Python error when performing a sync task. Any idea what's failing here?

I'm running Stash 0.21.0 in Docker, on an Ubuntu VM. Timestamp Trade is version 0.2. I'm installing `stashapp-tools` via:

`/usr/bin/docker exec -i "stash" /usr/bin/pip install stashapp-tools`

Here's the error I'm seeing. Thanks!

```
time="2023-08-10 06:44:43" level=info msg="[Plugin / Timestamp Trade] api returned something, for scene: A Cheer-Up Fuck marker count: 0"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/models.py\", line 971, in json"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     return complexjson.loads(self.text, **kwargs)"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/json/__init__.py\", line 346, in loads"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     return _default_decoder.decode(s)"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/json/decoder.py\", line 337, in decode"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     obj, end = self.raw_decode(s, idx=_w(s, 0).end())"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/json/decoder.py\", line 355, in raw_decode"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     raise JSONDecodeError(\"Expecting value\", s, err.value) from None"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade] json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade] During handling of the above exception, another exception occurred:"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 108, in <module>"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     processAll()"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 48, in processAll"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     processScene(s)"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 21, in processScene"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     md = res.json()"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]          ^^^^^^^^^^"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/models.py\", line 975, in json"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade]     raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)"
time="2023-08-10 06:44:51" level=error msg="[Plugin / Timestamp Trade] requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)"
time="2023-08-10 06:44:51" level=error msg="Plugin returned error: exit status 1"
```

**Comment by slabking** [comment]
I updated Stash to the newly released 0.22.0 and I think I'm seeing the same error:

```
time="2023-08-11 04:18:04" level=info msg="[Plugin / Timestamp Trade] api returned something, for scene: Last Night Of Freedom  marker count: 0"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/models.py\", line 971, in json"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     return complexjson.loads(self.text, **kwargs)"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/json/__init__.py\", line 346, in loads"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     return _default_decoder.decode(s)"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/json/decoder.py\", line 337, in decode"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     obj, end = self.raw_decode(s, idx=_w(s, 0).end())"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/json/decoder.py\", line 355, in raw_decode"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     raise JSONDecodeError(\"Expecting value\", s, err.value) from None"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade] json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade] During handling of the above exception, another exception occurred:"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 108, in <module>"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     processAll()"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 48, in processAll"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     processScene(s)"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 21, in processScene"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     md = res.json()"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]          ^^^^^^^^^^"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/models.py\", line 975, in json"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade]     raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)"
time="2023-08-11 04:19:10" level=error msg="[Plugin / Timestamp Trade] requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)"
time="2023-08-11 04:19:10" level=error msg="Plugin returned error: exit status 1"
```

**Comment by slabking** [comment]
I set the logging level to debug, but still no idea what's happening here. At least `/get-markers/54c6a568-19df-41ea-9246-160cd16ef9e0` likely points to a specific scene, though I haven't figured out the problematic scene yet.

```
time="2023-08-11 21:57:03" level=info msg="[Plugin / Timestamp Trade] api returned something, for scene: Last Night Of Freedom  marker count: 0"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/connection.py\", line 174, in _new_conn"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     conn = connection.create_connection("
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/util/connection.py\", line 72, in create_connection"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/socket.py\", line 962, in getaddrinfo"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] socket.gaierror: [Errno -3] Try again"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] During handling of the above exception, another exception occurred:"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 703, in urlopen"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     httplib_response = self._make_request("
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]                        ^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 386, in _make_request"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     self._validate_conn(conn)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 1042, in _validate_conn"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     conn.connect()"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/connection.py\", line 363, in connect"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     self.sock = conn = self._new_conn()"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]                        ^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/connection.py\", line 186, in _new_conn"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     raise NewConnectionError("
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7ff6689cbc10>: Failed to establish a new connection: [Errno -3] Try again"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] During handling of the above exception, another exception occurred:"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/adapters.py\", line 486, in send"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     resp = conn.urlopen("
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 787, in urlopen"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     retries = retries.increment("
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]               ^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/urllib3/util/retry.py\", line 592, in increment"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     raise MaxRetryError(_pool, url, error or ResponseError(cause))"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='timestamp.trade', port=443): Max retries exceeded with url: /get-markers/54c6a568-19df-41ea-9246-160cd16ef9e0 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff6689cbc10>: Failed to establish a new connection: [Errno -3] Try again'))"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] During handling of the above exception, another exception occurred:"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] Traceback (most recent call last):"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 108, in <module>"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     processAll()"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 48, in processAll"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     processScene(s)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/root/.stash/plugins/timestampTrade/timestampTrade.py\", line 20, in processScene"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     res = requests.post('https://timestamp.trade/get-markers/' + sid['stash_id'], json=s)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/api.py\", line 115, in post"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     return request(\"post\", url, data=data, json=json, **kwargs)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/api.py\", line 59, in request"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     return session.request(method=method, url=url, **kwargs)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/sessions.py\", line 589, in request"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     resp = self.send(prep, **send_kwargs)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/sessions.py\", line 703, in send"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     r = adapter.send(request, **kwargs)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]   File \"/usr/lib/python3.11/site-packages/requests/adapters.py\", line 519, in send"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade]     raise ConnectionError(e, request=request)"
time="2023-08-11 21:59:04" level=error msg="[Plugin / Timestamp Trade] requests.exceptions.ConnectionError: HTTPSConnectionPool(host='timestamp.trade', port=443): Max retries exceeded with url: /get-markers/54c6a568-19df-41ea-9246-160cd16ef9e0 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff6689cbc10>: Failed to establish a new connection: [Errno -3] Try again'))"
time="2023-08-11 21:59:05" level=error msg="Plugin returned error: exit status 1"
```
